ajax/libs/spritejs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "spritejs",
+  "version": "3.2.8",
+  "description": "A cross platform high-performance graphics system.",
+  "filename": "spritejs.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spritejs/spritejs.git"
+  },
+  "author": "akira-cn",
+  "homepage": "http://spritejs.org",
+  "keywords": [],
```

Comment:
keywords can be used for the search on the website, you can add the one spritejs uses

Reply:
OK, I taken the topic words into it which show in the github repo https://github.com/spritejs/spritejs, please check again, thank you.